Assets/Events/Threat.json

```
@@ -0,0 +1,174 @@
+﻿[
+  {
+    "name": "well-eye",
+    "headline": "Well Well Well, Look What Eye Have Here!",
+    "article": "A monstrous eye has been found living at the bottom of our local water hole? Our on the field reporter described peering into the well akin to \"Delving into the abyss itself... Where the true master of our world lies... Sorry, what were we talking about?\"",
+    "image": "behold",
+    "type": "Threat",
+    "outcomes": [
+      {
+        "type": "FlavourText",
+        "customDescription": "Your townsfolk start having the strangest dreams..."
+      },
+      {
+        "type": "ThreatAdded",
+        "baseAmount": 4
+      }
+    ]
+  },
+  {
+    "name": "time-being",
+    "headline": "Tributes to the Time Being",
+    "article": "A local cult has started making offerings to a mysterious creature, in exchange for future sight. \"We have a busy schedule ahead, so let's just leave this for the Time Being\" chant the cultists.",
+    "image": "empty-hourglass",
+    "type": "Threat",
+    "outcomes": [
+      {
+        "type": "FlavourText",
+        "customDescription": "Your townsfolk are fearful of this 'Time Being'"
+      },
+      {
+        "type": "ThreatAdded",
+        "baseAmount": 3
+      }
+    ]
+  },
+  {
+    "name": "beetles",
+    "headline": "Hello Beetles, Goodbye Beetles",
+    "article": "Giant monstrous beetles migrating from the strawberry fields terrorising the town this morning with loud melodic buzzing from the rooftops. Town etymologist had hoped they would have moved along yesterday. The blackbird population is reportedly under attack from these creatures. Local blacksmith Maxwell claims he has just the right tool for the situation...",
+    "image": "scarab-beetle",
```

```suggestion
    "headline": "Hello Beetles, Goodbye Beetles",
    "article": "Giant, monstrous beetles are migrating from the strawberry fields, terrorising the town this morning with loud, melodic buzzing from the rooftops. The town etymologist had hoped they would have moved along yesterday. The blackbird population is reportedly under attack from these creatures. Local blacksmith, Maxwell, claims he has just the right tool for the situation...",
    "image": "scarab-beetle",
```

yuff this is a lot to pack in. Doesn't read super nicely.